slides/03_workflow.qmd

```diff
@@ -1,10 +1,591 @@
 ---
-title: "A Package Engineering Workflow"
+title: "An R Package Engineering Workflow"
+subtitle: "BBS Course: Good Software Engineering Practice with R"
 author: "Friedrich"
+affiliation: "RPACT"
+institute: "RPACT"
+date: '2023-02-10'
+date-format: long
+tags: [R, Package]
+license: 'CC BY-SA'
+email-obfuscation: javascript
 format: 
-  revealjs
-description: description. Photo CC0 by Andrea Piacquadio on pexels.com.
+  revealjs:
+    theme: [default, custom.scss]
+    fontsize: 34px
+    toc: false
+    slide-number: true
+    progress: true
+    code-line-numbers: true
+    mouse-wheel: false
+    link-external-newwindow: true
+    footer: 'Good Software Engineering Practice with R - [Home](../index.html "GSWEP4R Workshop") | [Slides](listing.html "Overview of all slides") | [License](http://creativecommons.org/licenses/by-sa/4.0/ "License: CC BY-SA 4.0")'
+description: "Workflow for creating R packages + practical<br/>[Photo CC0 by Andrea Piacquadio on pexels.com]{.copyright}"
 image: thumbnails/workflow.jpg
+lang: en
+editor: 
+  markdown: 
+    wrap: 80
 ---
 
-scripts / function / design docs / packaging, orga
+# [{{< fa face-grin-stars >}}]{.yellow} Motivation
+
+```{=html}
+<!-- 
+- One-off scripts, prototype functions
+- R generic functions
+- Design docs
+- Packaging
+- Professional workflow
+-->
+```
+**From an idea to a production-grade R package**
+
+Example szenario: in your daily work, you notice that you need certain one-off
+scripts again and again.
+
+. . .
+
+[{{< fa lightbulb size=2x >}}]{.blue} The idea of creating an R package was born
```

Comment:
this is really cool with these icons! I guess I will try to find some time to beef up my slides  too